moveit_demo_nodes/run_move_group/launch/run_move_group.launch.py

```diff
@@ -124,11 +48,11 @@ def generate_launch_description():
         output="log",
         arguments=["-d", rviz_config_file],
         parameters=[
-            robot_description,
-            robot_description_semantic,
-            ompl_planning_pipeline_config,
-            kinematics_yaml,
-            joint_limits_yaml,
+            moveit_config.robot_description,
+            moveit_config.robot_description_semantic,
+            moveit_config.robot_description_kinematics,
+            moveit_config.planning_pipelines,
+            moveit_config.joint_limits,
```

Comment:
Above comment about doing `moveit_config` -> list wouldn't help here if we only want to send some of the parameters. (Side note: is there a downside passing all the moveit config parameters here?)